pkg/proc/breakpoints.go

```diff
@@ -113,6 +115,11 @@ type Breaklet struct {
 	watchpoint *Breakpoint
 }
 
+// SetCallBack sets the call back field, this was primarily added to prevent exporting callback field
+func (B *Breaklet) SetCallBack(callback DynCallFn) {
```

Comment:
We don't use capital letters for local variable names. Also the name `DynCallFn` is odd for this and personally I'd just write the inlined type here.